Makefile

```diff
@@ -1,17 +1,12 @@
-NAME    := cucumber/cucumber-build
-VERSION := 0.2.0
+NAME      := cucumber/cucumber-build
+VERSION   := 0.2.0
```

Comment:
Why is this needed? Can we just omit the `—platforms` flag if the env var isn’t set?